mypy/stubgen.py

```diff
@@ -84,11 +88,18 @@ def generate_stub_for_module(module: str, output_dir: str, quiet: bool = False,
                              search_path: List[str] = [],
                              interpreter: str = sys.executable) -> None:
     target = module.replace('.', '/')
-    result = find_module_path_and_all(module=module,
-                                      pyversion=pyversion,
-                                      no_import=no_import,
-                                      search_path=search_path,
-                                      interpreter=interpreter)
+    try:
+        result = find_module_path_and_all(module=module,
+                                          pyversion=pyversion,
+                                          no_import=no_import,
+                                          search_path=search_path,
+                                          interpreter=interpreter)
+    except CantImport:
+        if not quiet:
```

Comment:
I made it print the error message either way. I noticed though that quiet mode is unused; it defaults to False and nothing outside of tests calls generate_stub_for_module with quiet=True.